src/Raven.Client/Documents/Session/ISessionDocumentTimeSeriesBase.cs

```diff
@@ -15,19 +15,24 @@ namespace Raven.Client.Documents.Session
     public interface ISessionDocumentTimeSeriesBase
     {
         /// <summary>
-        /// Append the the values (and tag) to the times series at the provided timstamp
+        /// Append the the values (and optional tag) to the times series at the provided time stamp
         /// </summary>
-        void Append(string timeseries, DateTime timestamp, string tag, IEnumerable<double> values);
+        void Append(DateTime timestamp, IEnumerable<double> values, string tag = null);
 
         /// <summary>
-        /// Remove all the values in the timeseries in the range of from .. to.
+        /// Append a single value (and optional tag) to the times series at the provided time stamp
         /// </summary>
-        void Remove(string timeseries, DateTime from, DateTime to);
+        void Append(DateTime timestamp, double value, string tag = null);
```

Comment:
What do you think about `Append` without `DateTime` something like `AppendNow`? Do we want something like that?
It can use either server or client time.

Reply:
No, exactly because this is so hard to figure out, let's do that explicitly. 
User need to call `DateTime.UtcNow`, not really hard

Reply:
Just to note that if we put server's time we avoid conflicts on the the time-stamp and can handle clock/time issues.
But I guess if needed we can add this later.

Reply:
Not really a good idea. Because now you added network lag to the time we measure.